Màj données TGV

martinregner opened the pull request:
Nouvelles données Pierre

martinregner committed:
Màj données TGV

Nouvelles données Pierre

laem commented:
Pour ajouter des emojis aux commits, voilà la liste : https://gist.github.com/rxaviers/7360908 ;)

martinregner committed:
Màj FE et sources

martinregner committed:
Ajout de nouveaux modes

laem committed:
Petit problème d'indentation du YAML

laem committed:
Simplification des catégories de motos

J'ai pris ce tableau en compte :
https://www.auto-ecole.net/img/banque_image/equivalence_moto1499185981.jpg

laem committed:
Renommage tram et données simplifiées

laem committed:
Modes au singulier

laem commented:
@martinregner je me suis permi de donner des titres aux motos plutôt que "deux roues < 250cm3", parce qu'à mon avis ça ne parle qu'aux experts

- [x] @martinregner je tique juste sur la voiture. Dans le modèle d'avant, on donnait un facteur / personne. Ici, je crois qu'on revient sur un facteur global, indifférencié, de la voiture. Il faut donc que je modifie le calcul pour inverser le calcul.
- [x] élargir la zone des émojis pour en accuillir 2 sur la même ligne

laem committed:
:art: Tentative d'alignement des lignes à deux emojis

laem committed:
Evolution du calcul pour la voiture

laem committed:
:art: Possibilité d'ajouter un emoji secondaire

Pour les modes électriques

laem committed:
:art: On enlève la marge sur bureau entre classement et saisie

laem committed:
On passe de version test à beta; lien de contact en bas

laem committed:
Typo deux roues
